File: Backend/server/Models/Ticket.cs
using System;
using System.Collections.Generic;

namespace ServicesGatway.Models;

public partial class Ticket
{
    public int TicketId { get; set; }

    public int FlightId { get; set; }

    public int CustomerId { get; set; }

    public DateTime PurchaseDate { get; set; }

    //public virtual Customer Customer { get; set; } = null!;

    //public virtual Flight Flight { get; set; } = null!;
}

File: Frontend/Model/Ticket.py
class Ticket:
    def __init__(self, ticket_id, flight_id, customer_id, purchase_date):
        self.ticket_id = ticket_id
        self.flight_id = flight_id
        self.customer_id = customer_id
        self.purchase_date = purchase_date

    def __repr__(self):
        return (f"Ticket(ID={self.ticket_id}, FlightID={self.flight_id}, CustomerID={self.customer_id}, "
                f"PurchaseDate={self.purchase_date})")
